ex1/functions.py

```diff
@@ -90,26 +135,98 @@ def f():
 # Define a function named args_to_dict that takes three arguments and returns a
 # dictionary with the position of the argument as the key (starting at 0) and
 # the argument as the value.
-
+def args_to_dict(arg1, arg2, arg3):
+    loc = locals()    
+    kwargs = dict()
+    i = 0    
+    for arg in loc:
+        kwargs[i]=loc.get(arg)
+        i+=1
+    return kwargs
+    
 # BONUS: Write a function named args_to_dict_general that does the same for any
 # number of arguments
+def args_to_dict_general(*args):
+    loc = locals()    
+    kwargs = dict()
+    i = 0    
+    for args in loc:
+        for arg in args:
+            kwargs[i]=arg
+            i+=1
+    return kwargs
+
 
 # Define a function named lists_to_dict that takes two lists of equal lenght
 # named keys and values and builds a dictionary out of them.
-
+def lists_to_dict(list1, list2):
+    d=dict()
```

Comment:
I don't understand what you are trying to do here. See [the solution](https://github.com/TUW-GEO-python-intro/Exercise-1/pull/20/files#diff-7ec7b04e6cd2fbdbc70b7b64bf731ae8R339) for a simpler approach using `zip`